packages/amplify-category-geo/src/commands/geo/update.ts

```diff
@@ -20,8 +20,9 @@ export const run = async(context: $TSContext) => {
     return await updateResource(context, result.service);
 
   } catch (error) {
+    error.message ? context.print.error(error.message) : '';
```

Comment:
seems odd to use a ternary for a conditional action. Maybe it would be better to just use an if statement here?
```
if (error.message) {
  context.print.error(error.message)
}
```

(applies elsewhere as well)